addons/account/tests/test_account_move_in_invoice.py

```diff
@@ -1478,9 +1478,10 @@ def test_in_invoice_duplicate_supplier_reference(self):
         ''' Ensure two vendor bills can't share the same vendor reference. '''
         self.invoice.ref = 'a supplier reference'
         invoice2 = self.invoice.copy(default={'invoice_date': self.invoice.invoice_date})
-
+        # changing the ref shouldn't trigger an error
```

Comment:
But posting should. How should I improve this comment? (Or should I just delete it?)

Reply:
You can just remove the comment, the test is small and explicit enough